src/main/java/org/takes/rq/RqHeaders.java

```diff
@@ -49,6 +49,7 @@
  * @version $Id$
  * @since 0.1
  */
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
@erimerturk this file is really large now. Let's add a puzzle to extract `RqHeaders.Smart` into its own file and then remove this suppression.